iothub/device/src/DeviceClient.cs

```diff
@@ -325,6 +325,18 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         /// <returns>The receive message or null if there was no message until the specified time has elapsed</returns>
         public Task<Message> ReceiveAsync(TimeSpan timeout) => InternalClient.ReceiveAsync(timeout);
 
+        /// <summary>
+        /// Registers a new delegate for receiving a message from the device queue using the default timeout.
+        /// After handling a received message, a client should call <see cref="CompleteAsync(Message, CancellationToken)"/>,
```

Comment:
Add documentation on how to unset this callback.

Reply:
It is present on L332